ui/apps/profile/src/components/pages/profile-info-page.tsx

```diff
@@ -85,7 +85,7 @@ const ProfileInfoPage: React.FC<RootComponentProps> = props => {
 
   const handleCopy = () => {
     /** handle copy */
-    const profileUrl = `${location.origin}/@akashaorg/app-profile/${profileId}`;
+    const profileUrl = `${location.origin}${location.pathname}`;
```

Comment:
instead of a string, you could build an actual `URL` object with this arguments, and afterwards pass the `href` property of the object.